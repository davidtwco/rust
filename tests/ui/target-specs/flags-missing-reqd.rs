//@ compile-flags: -Ztarget-spec=aarch="aarch64" --target=aarch64-flags-flags
//@ error-pattern: Error loading target specification: Field llvm-target in target specification is required.
//@ needs-llvm-components:
