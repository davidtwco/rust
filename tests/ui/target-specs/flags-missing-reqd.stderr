error: Error loading target specification: Field llvm-target in target specification is required. Run `rustc --print target-list` for a list of built-in targets

